### DISCUSSION TITLE
From One Lifelong TALES fan To Another

### DISCUSSION THREAD

**Discussion by lordsirec** [furiousg4m3r/Tales-of-Destiny-DC#1675] [General]
I just wanted to take the time to express just how the simple fact that you guys are still actively working on this in 2022 makes my heart smile. As I dive headlong into this project of yours with all the joy and enthusiasm of a child on christmas morning, i will be sure to keep an eye out for any possible bugs or untranslated tidbits for you guys(if my playthrough of this benefits you guys and this project in any way I'm happy to help!!). There are so many TALES titles that sadly never left Japan(which for those who cant read japanese like me is a sad thing indeed) So seeing projects like this makes me extremely happy and SUPER EXCITED!! Keep up the amazing work!!

**Comment by pnvnd** [comment]
```
 _     _  __        ____        _   _   _      
| |   (_)/ _| ___  | __ )  ___ | |_| |_| | ___ 
| |   | | |_ / _ \ |  _ \ / _ \| __| __| |/ _ \
| |___| |  _|  __/ | |_) | (_) | |_| |_| |  __/
|_____|_|_|  \___| |____/ \___/ \__|\__|_|\___|
```
